packages/components/src/List/ListComposition/Manager/filter.hook.js

```diff
@@ -0,0 +1,40 @@
+import { useMemo, useState } from 'react';
+
+function defaultFilterFunction(value, textFilter) {
+	return value
```

Comment:
need to test if the value is not null or undefined

Reply:
Set a default value to ''